README.rst

```diff
@@ -99,6 +99,8 @@ GNU General Public License v3.0
 See `COPYING <COPYING>`_ to see the full text.
 
 .. |PyPI version| image:: https://img.shields.io/pypi/v/ansible.svg
-   :target: https://pypi.python.org/pypi/ansible
+   :target: https://pypi.org/project/ansible
+.. |Docs badge| image:: https://img.shields.io/badge/docs-latest-brightgreen.svg
```

Comment:
This seems to be a static image, is that correct?

Reply:
Oh, just so we have a more visible link to the docs?

Reply:
Yeah.  A more visible link that fits in with the theme of the other badges.